spec/features/inviting_new_submissions_in_place_of_expired_ones_feature_spec.rb

```diff
@@ -0,0 +1,40 @@
+# Testing rake tasks method found at:
+# http://carlosplusplus.github.io/blog/2014/02/01/testing-rake-tasks-with-rspec/
+# https://robots.thoughtbot.com/test-rake-tasks-like-a-boss
+
+require 'rails_helper'
+require 'rake'
+
+describe 'inviting new submissions in place of expired ones' do
```

Comment:
I would like to live in the world where running your test suite is all you need to catch regressions. Without those tests, it's easy to forget about updating rake tasks when service's interface changes. I see two options for improvement here:
1. The config can be extracted to shared example and will work for each rake task spec.
2. I can remove those tests completely because when changing interface you should probably grep the repo anyway to double check.
What do you think?